exercises/simple-linked-list/simple_linked_list_test.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+from example import LinkedList, EmptyListException
+
+
+# No canonical data available for this exercise
+
+class LinkedListTest(unittest.TestCase):
+    def test_empty_list_has_len_zero(self):
+        sut = LinkedList()
```

Comment:
What does `sut` mean? I would prefer a variable name with a clear meaning - `linked_list` would work.

Reply:
`sut` is System Under Test.